pkg/kubelet/kubelet.go

```diff
@@ -1747,8 +1747,8 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 		go func() {
 			// Call updateRuntimeUp once before syncNodeStatus to make sure kubelet had already checked runtime state
 			// otherwise when restart kubelet, syncNodeStatus will report node notReady in first report period
-			kl.updateRuntimeUp()
-			wait.JitterUntil(kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, 0.04, true, wait.NeverStop)
+			kl.updateRuntimeUp(ctx)
+			go wait.JitterUntilWithContext(ctx, kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, 0.04, true)
```

Comment:
We don't need to add the **go** keyword here. Please keep the original logic and only modify `JitterUntil` to `JitterUntilWithContext`.

Reply:
done thank you.